backend/src/service/Jimmi.service.ts

```diff
@@ -22,7 +22,7 @@ export function getAllJimmiIds(): string[] {
 
 /**
  * Returns Jimmi instance with the given id
- * @param id id of Jimmi instance
+ * @param id id of the Jimmi instance
```

Comment:
```suggestion
 * @param id - id of the Jimmi instance
```
To conform to the TSDoc spec